lib/flags.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+class Flags {
+  static from(...args) {
+    const values = args;
+    class FlagsClass {
+      static from(...args) {
+        return new FlagsClass(...args);
+      }
+
+      constructor(...args) {
+        this.values = new Map(values.map(v => [v, 0]));
```

Comment:
I think the idea was to use Int of another sort of BitPacking in order to store such set values.

Reply:
Yes, but we need to convert `key` to position of the bit which means that need either to use `Map` or to iterate over array to find this position. There won't be need in `Map`s at all if there was indexes of bits instead of its names.

Reply:
Yes, but I think the idea was to have cheap usage (number is already created in the flags instance so `toNumber` is just `return number`) and not so cheap value setting (find value index (we can also use Map for this, no need for linear search) -> set bit at that index).